pom.xml

```diff
@@ -3,18 +3,54 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.activiti</groupId>
+    <groupId>org.activiti.build</groupId>
     <artifactId>activiti-parent</artifactId>
-    <!--<version>7.0.6</version>--> <!-- using fixed version for CI/CD -->
-    <version>7.0.0-SNAPSHOT</version>
+    <version>7.0.0-SNAPSHOT</version> <!-- using fixed version for CI/CD -->
+    <relativePath/>
   </parent>
   <groupId>org.activiti.api</groupId>
   <artifactId>activiti-api</artifactId>
   <version>7.0.0-SNAPSHOT</version>
 
-  <name>Activiti :: Runtime API Parent</name>
+  <name>Activiti API :: Parent</name>
   <packaging>pom</packaging>
+  <dependencyManagement>
+    <!-- This duplicates activiti-api-dependencies module but we can't import that module here as this is its parent-->
```

Comment:
Spring solves that by having a root pom plus a parent at the same level as the other modules. But we can improve that later on.